### PR TITLE
Update football network to parametric action distribution API

### PR DESCRIPTION
**Pull request by CStanKonrad**
Now football network uses parametric action distribution to sample and postprocess actions.

### CODE REVIEW

File: football/networks.py [hunk 1]
```diff
@@ -132,6 +132,11 @@ def __call__(self, input_, core_state, unroll=False,
       # Remove time dimension.
       outputs = tf.nest.map_structure(lambda t: tf.squeeze(t, 0), outputs)
 
+    if not is_training:
+      outputs = AgentOutput(
```
Comment (lespeholt) [inline football/networks.py]: outputs = outputs._replace(action=...)

**Comment by lespeholt** [comment]
Looks good. Added a comment.

**Comment by CStanKonrad** [comment]
Should I also change 
https://github.com/google-research/seed_rl/blob/8af694ebd21af5f1b14bd8891e84aa42c5f13b2f/agents/vtrace/networks.py#L75
in order to maintain consistency.

**Comment by lespeholt** [comment]
That would be great thanks

**Comment by lespeholt** [comment]
Thanks

**Pull request merged**